lib/models/list.js

```diff
@@ -19,6 +29,23 @@ async function readList(id) {
   }
   return list;
 }
+
+async function updateProductsInList(id, product) {
+  const listCollection = await getCollection("lists");
+  const objectId = new ObjectID.createFromHexString(id);
```

Comment:
I would call it `ObjectId` to keep the naming consistent.

Reply:
but here it's a new const which I created, therefore it will be written in camelCase, or?